### PR TITLE
feat: leveldb indexer usage doc

### PR DESCRIPTION
**Pull request by rutujasurve94**
Doc for `LevelDB` indexer usage.

### CODE REVIEW

**Review by bwanglzu — DISMISSED** [review]
LGTM!

File: indexers/keyvalue/LevelDBIndexer/README.md [hunk 1]
```diff
@@ -1,3 +1,73 @@
...
+  
+- YAML file
+
+  This is the only way to provide arguments to its parameters:
+  
+  ```yaml
+  pods:
+    - name: leveldb
+      uses: docker://jinahub/pod.indexer.leveldbindexer:MODULE_VERSION-JINA_VERSION 
+      uses_internal: leveldb.yml
```
Comment (deepankarm) [inline indexers/keyvalue/LevelDBIndexer/README.md:62]: Why do we have `uses` fetch from a docker & `uses_internal` fetch from yml here? Let's only have the yml?

File: indexers/keyvalue/LevelDBIndexer/README.md [hunk 2]
```diff
@@ -1,3 +1,73 @@
...
+  ```bash
+  jina pod --uses hub/example/config.yml --port-in 55555 --port-out 55556
+  ```
+
+- Docker command
+
+  Specify the image name along with the version tag. The snippet below uses Jina version as `JINA_VERSION`.
+
+  ```bash
+  docker pull jinahub/pod.indexer.leveldbindexer:MODULE_VERSION-JINA_VERSION
```
Comment (deepankarm) [inline indexers/keyvalue/LevelDBIndexer/README.md:51]: `docker pull` & `docker run` both are listed here. We don't need both,

File: indexers/keyvalue/LevelDBIndexer/manifest.yml [hunk 3]
```diff
@@ -7,6 +7,6 @@ author: Jina AI Dev-Team (dev-team@jina.ai)
 url: https://jina.ai
 vendor: Jina AI Limited
 documentation: https://github.com/jina-ai/jina-hub
```
Comment (deepankarm) [inline indexers/keyvalue/LevelDBIndexer/manifest.yml:9]: Please point to the readme of the master branch

**Review by deepankarm — CHANGES_REQUESTED** [review]

**Comment by deepankarm** [comment]
As discussed, let's follow a single standard for `pod usage` in all executor documentations,

**Comment by rutujasurve94** [comment]
@deepankarm I've updated the README as your suggestions, thanks a lot for the review.

**Review by deepankarm — APPROVED** [review]

**Pull request merged**